arm/Microsoft.KeyVault/vaults/keys/deploy.bicep

```diff
@@ -57,8 +57,13 @@ module pid_cuaId './.bicep/nested_cuaId.bicep' = if (!empty(cuaId)) {
   params: {}
 }
 
+resource keyVault 'Microsoft.KeyVault/vaults@2021-06-01-preview' existing = {
+  name: vaultName
```

Comment:
I'd suggest `keyVaultName` instead of `vaultName` for the parent resource name in all children

Reply:
done